Makefile

```diff
@@ -121,10 +121,6 @@ $(D):
 $(I): $(O)
 	$(Q)mkdir -p $(I)
 
-mrproper:
```

Comment:
ok - what is the alternative here?

Reply:
Run regular 'make mrproper' on each subproject, then `rm` only the generated files in the top level. No `git clean -fdx` needed.

Reply:
Sure, could you update the commit to do the right thing instead of dropping the mrproper rule?